src/entities/SurveyTopic/utils.ts

```diff
@@ -0,0 +1,23 @@
+import { ReactionType, Survey } from './types'
+
+type ReactionView = { reaction: ReactionType; icon: string }
+export const REACTIONS_VIEW: ReactionView[] = [
```

Comment:
Why is it called "view"? Maybe `REACTIONS_LIST`?

Reply:
Changed to

```typescript
type ReactionWithIcon = { reaction: Reaction; icon: string }
export const REACTION_LIST: ReactionWithIcon[] = [
  { reaction: Reaction.LOVE, icon: '🥰' },
  { reaction: Reaction.LIKE, icon: '😊' },
  { reaction: Reaction.NEUTRAL, icon: '😐' },
  { reaction: Reaction.CONCERNED, icon: '🤨' },
  { reaction: Reaction.EMPTY, icon: '-' },
]
```